Driverless AI/natural-language-processing-tutorial-sentiment-analysis /.ipynb_checkpoints/natural-language-processing-tutorial-sentiment-analysis-checkpoint.md

```diff
@@ -425,90 +416,88 @@ A Bi-directional GRU model is like putting two independent RNN models in one. Ta
 
 7. **Convolution neural network models on:**
 
-- **word embeddings**
+-   **word embeddings**
 
 In Driverless AI, we pass word embeddings as input to CNN models, get cross-validated predictions from it and use them as a new set of features.
 
--  **CNN models on character embeddings**
+-   **CNN models on character embeddings**
 
 Natural language processing is complex as the language is hard to understand given small data and different languages.Targeting languages like Japanese, Chinese where characters play a major role, we have character level embeddings in our recipe as well.
 
 In character embeddings, each character gets represented in the form of vectors rather than words. Driverless AI uses character level embeddings as input to CNN models and later extract class probabilities to feed as features for downstream models. This gives the ability to work in languages other than English also. In case of languages like Japanese, Chinese etc where there is no concept of words, character embeddings will be useful.
```

Comment:
@Castro38 rewrite the paragraph to this: 

"In character embeddings, each character gets represented in the form of vectors rather than words. Driverless AI uses character level embeddings as input to CNN models and later extracts class probabilities to feed as features for downstream models: this gives the ability to work in languages other than English. In languages like Japanese and Chinese, where there is no concept of words, character embeddings will be useful."

Reply:
@5675sp Done